addon/initializers/m3-store.js

```diff
@@ -87,6 +88,16 @@ const STORE_OVERRIDES = {
     return this._queryCache.contains(cacheKey);
   },
 
+  // override _push to batch change notifications which we're obliged to do
+  // since all our properties are treated as volatiles as they come from
+  // `unknownProperty`
+  _push(jsonApiDoc) {
+    beginChanges(this);
```

Comment:
Is `beginChanges` unnecessary given that `deferChanges` does that?

Reply:
fair enough; i meant for it to better signal intent, but i'll drop it